DataFormats/EcalRecHitSoA/interface/EcalUncalibratedRecHit_soa.h

```diff
@@ -0,0 +1,74 @@
+#ifndef DataFormats_EcalRecHitSoA_interface_EcalUncalibratedRecHit_soa_h
+#define DataFormats_EcalRecHitSoA_interface_EcalUncalibratedRecHit_soa_h
+
+#include <vector>
+#include <array>
+
+#include "DataFormats/EcalDigi/interface/EcalDataFrame.h"
+#include "DataFormats/EcalRecHitSoA/interface/RecoTypes.h"
+
+#include "HeterogeneousCore/CUDAUtilities/interface/CUDAHostAllocator.h"
```

Comment:
Data formats depending on any CUDA stuff should be placed under `CUDADataFormats` (so `CUDADataFormats/EcalRecHit`). For non-CUDA dependent SOA I'd suggest the existing data format package (i.e. `DataFormats/EcalRecHit`).